hadoop-hdds/docs/themes/ozonedoc/static/swagger-resources/recon-api.yaml

```diff
@@ -0,0 +1,1584 @@
+    description: APIs to fetch data about the cluster state
+  - name: Datanodes
+    description: APIs to fetch data about the Datanodes
+  - name: Pipelines
+    description: APIs to fetch data about the Pipelines
+  - name: Tasks
+    description: APIs to fetch data about status of Recon Tasks
+  - name: Utilization
+    description: APIs to fetch data about space utilization
+  - name: Metrics
+    description: APIs to fetch data about various metrics from Prometheus
+    externalDocs:
+      description: Prometheus API docs
+      url: https://prometheus.io/docs/prometheus/latest/querying/api/
+paths:
+  /containers:
+    get:
+      tags:
+        - Containers and Keys
+      summary: Get all Container Metadata information
+      operationId: getContainerInfo
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ContainerMetadata'
+  /containers/{containerId}/keys:
+    get:
+      tags:
+        - Containers and Keys
+      summary: Get the Key metadata for all keys present in the given container ID
+      operationId: getContainerKeyMetadata
+      parameters:
+        - name: containerId
+          in: path
+          description: ID of the container for which we want Key Metadata
+          required: true
+          schema:
+            type: integer
+        - name: prevKey
+          in: query
+          description: Only return keys that are present after the given key prevKey prefix
+          required: false
+          schema:
+            type: string
+            example: /vol1/bucket1/key1
+        - name: limit
+          in: query
+          description: Limit of the number of results returned
+          required: false
+          schema:
+            type: integer
+            default: 1000
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/KeyMetadata'
+  /containers/missing:
+    get:
+      tags:
+        - Containers and Keys
+      summary: Get the MissingContainerMetadata for all missing containers
+      operationId: getMissingContainers
+      parameters:
+        - name: limit
+          in: query
+          description: Limit of the number of results returned
+          required: false
+          schema:
+            type: integer
+            default: 1000
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/MissingContainerMetadata'
+  /containers/{containerId}/replicaHistory:
+    get:
+      tags:
+        - Containers and Keys
+      summary: Get all the Replica history for a given container id
+      operationId: getReplicaHistory
+      parameters:
+        - name: containerId
```

Comment:
Pls check this param name. Seems not correct.

Reply:
This is the same as https://github.com/apache/ozone/pull/5509#discussion_r1381910275

Reply:
Reverted to {id} in the latest commit